server/src/test/java/org/elasticsearch/search/aggregations/bucket/geogrid/GeoGridAggregatorTestCase.java

```diff
@@ -240,4 +243,14 @@ private void testCase(Query query, int precision, GeoBoundingBox geoBoundingBox,
         indexReader.close();
         directory.close();
     }
+
+    @Override
+    protected AggregationBuilder createAggBuilderForTypeTest(MappedFieldType fieldType, String fieldName) {
+        return createBuilder("foo").field(fieldName);
+    }
+
+    @Override
+    protected List<ValuesSourceType> getSupportedValuesSourceTypes() {
```

Comment:
My preference is to PR these tests separately against master first, so we can validate that the type support hasn't changed in the registry branch.

Reply:
will do this in a separate PR

Reply:
opened https://github.com/elastic/elasticsearch/pull/53180